model/src/pyrenew/latent/infections.py

```diff
@@ -104,8 +106,17 @@ def sample(
 
         gen_int_rev = jnp.flip(gen_int)
 
-        n_lead = gen_int_rev.size - 1
-        I0_vec = jnp.hstack([jnp.zeros(n_lead), I0])
+        if I0.size < gen_int_rev.size:
+            raise ValueError(
+                "Initial infections vector must be at least as long as "
+                "the generation interval."
```

Comment:
```suggestion
                "the generation interval. "
```
Fix spacing for single-line printing.